tests/php/ExternalLinks/ExternalLinksTest.php

```diff
@@ -125,6 +127,35 @@ public function testArchivedPagesAreHiddenFromReport()
         $this->assertEquals(3, BrokenExternalLinksReport::create()->sourceRecords()->count());
     }
 
+    public function testMissingPageGracefullyHandled(): void
+    {
+        // Create an output buffer to capture the output
+        $bufferedOutput = new BufferedOutput();
+        $polyOutput = new PolyOutput(PolyOutput::FORMAT_ANSI, PolyOutput::VERBOSITY_NORMAL, false, $bufferedOutput);
+        // Run link checker to generate BrokenExternalPageTrackStatus
+        // This needs to be created with ::create() so that its private static $dependencies are injected
+        $task = CheckExternalLinksTask::create();
```

Comment:
```suggestion
        $task = new CheckExternalLinksTask();
```
Should be using `new` keyword as per https://github.com/silverstripe/silverstripe-reports/pull/210#discussion_r2015294520

Reply:
There's a comment above about the intentional use of ::create()

Reply:
Ahh sorry, missed that.

Reply:
We could instantiate a new links checker and pass it in but not worth the extra ping pong probably.